dist/github-actions/.github/workflows/ci.yml

```diff
@@ -105,3 +105,38 @@ jobs:
         run: |
           vendor/bin/robo job:serve-drupal &
           vendor/bin/robo job:behat-tests
+
+  cypress:
+    runs-on: ubuntu-latest
+    container:
+      image: juampynr/drupal8ci:latest
```

Comment:
Does this need the drupal8 image or could it use the drupal9 one?

Reply:
It depends on the PHP version requirements. For projects using PHP7.3 or PHP7.4, the `drupal8ci` CI image should be used. For projects using PHP8.0 then the `drupal9ci` is good.